WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -345,12 +348,28 @@ private async Task CreateRoundsAsync(CancellationToken cancellationToken)
 		{
 			var feeRate = (await Rpc.EstimateSmartFeeAsync((int)Config.ConfirmationTarget, EstimateSmartFeeMode.Conservative, simulateIfRegTest: true, cancellationToken).ConfigureAwait(false)).FeeRate;
 
-			RoundParameters roundParams = new(Config, Network, Random, feeRate, Config.CoordinationFeeRate);
+			RoundParameters roundParams = new(Config, Network, Random, feeRate, Config.CoordinationFeeRate, GetMaxSuggestedAmount(ConnectionConfirmationStartedCounter));
 			Round r = new(roundParams);
 			Rounds.Add(r);
 		}
 	}
 
+	internal static Money GetMaxSuggestedAmount(int roundCounter)
+	{
+		Money baseAmount = Money.Coins(0.1m);
```

Comment:
Maybe smaller? Not sure.
```suggestion
		Money baseAmount = Money.Coins(0.01m);
```

Reply:
Added to WabiSabiConfig as a parameter. the current default is 0.1 still